lib/libyui_client/logger.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'forwardable'
+require 'logger'
+
+module LibyuiClient
+  class Logger
+    extend Forwardable
+
+    def_delegators :@logger, :debug, :debug?,
+                   :info, :info?,
+                   :warn, :warn?,
+                   :error, :error?,
+                   :fatal, :fatal?,
+                   :level
+
+    def initialize
+      @logger = create_logger($stdout)
+    end
+
+    def create_logger(output)
```

Comment:
let's call it in the same way both, `$stdout` or `output`, wdyt?

Reply:
thanks for the clarification, I got confused with Perl somehow with the dollar symbol :)